draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -854,9 +854,20 @@ cookie        = cookie-string
 cookie-string = cookie-pair *( ";" SP cookie-pair )
 ~~~
 
+While {{Section 5.4 of RFC9110}} does not define a length limit for header
+fields it is likely that the web server's implementation does impose a limit;
+many popular implementations have default limits of 8K. Server SHOULD avoid
+setting a large number of large cookies such that the final cookie-string
+would exceed their header field limit. Not doing so could result in requests
+to the server failing.
+
 Servers MUST be tolerant of multiple cookie headers. For example, an HTTP/2
 {{RFC9113}} or HTTP/3 {{RFC9114}} connection might split a cookie header to
-improve compression.
+improve compression. Servers are free to determine what form this tolerance
```

Comment:
Commenting on the line above, the `connection` doesn't do the splitting per se. Instead, the protocol allows the user agent to split the cookie. As a server you have to accept the split cookie when processing, and then move on to figuring out your tolerance policy (as you're covering in the new text)

Reply:
Are you asking that the phrase `...connection might split a cookie header...` be changed?

Reply:
Perhaps s/connection/client or intermediary/?

Reply:
Sounds good to me